recipe/meta.yaml

```diff
@@ -29,6 +29,7 @@ build:
     - libstdcxx-ng
   missing_dso_whitelist:
     - "*/libc++.1.dylib" # [osx]
+    - $RPATH/libstdc++.so   # [linux]
```

Comment:
This DSO should not be detected as missing; based on the comments further up in the recipe, it should be statically linked in so `conda build` should not be identifying any binaries that need it. Having to inject `libstdc++.so` into the `missing_dso_whitelist` is an indication that something else in the recipe has broken.

Reply:
There is no static support for libstdc++ by default on CF nor on our g++.  Mainly caused by the fact that sysroot comes without a static glibc either.  There is already an issue on CF's sysroot to add static glibc support too.
Right now static linking of CF's and our gcc is (by intention) not working

Reply:
Indeed the comment ' # libc++/libstdc++: statically linked' is misleading ...

Reply:
and the line should be '- $RPATH/$RPATH/libstdc++.so.6' ... you've missed the .6 postfix